### ISSUE TITLE
WDM is unable to download the 64-bit IEDriverServer

### ISSUE THREAD

**Issue by djangofan** [bonigarcia/webdrivermanager#19]
WDM is unable to download the 64-bit IEDriverServer.   The filter that occurs on BrowserManager.java, line 219 to 226 (labeled 'Round 1 : Filter by OS'),  is filtering out the IEDriverServer url that contains the string 'x64' instead of 'Win' .   The 'url.getFile().toLowerCase().contains never will match because the string doesn't contain 'Win' in it.     

Example URL: 

> http://selenium-release.storage.googleapis.com/2.52/IEDriverServer_x64_2.52.0.zip

Here is the method that is having the problem: 
```
		for (URL url : list) {
			for (OperativeSystem os : OperativeSystem.values()) {
				if (MY_OS_NAME.contains(os.name())
						&& url.getFile().toLowerCase().contains(os.name())) {
					out.add(url);
				}
			}
		}
```

**Comment by bonigarcia** [comment]
Thanks for reporting. I'll look into this issue ASAP.

**Comment by bonigarcia** [comment]
Solved in version 1.4.1.

**Comment by Evleky** [comment]
This still seems to be an issue. It looks like getDriverName().equals("IEDriverServer") was added to try and catch this, however, getDriverName() returns a list, thus will never .equals a String. So it is still never finding 64 bit IEDriverServer.

Offending snippet:
`		for (URL url : list) {
			for (OperativeSystem os : OperativeSystem.values()) {
				if (((MY_OS_NAME.contains(os.name())
						&& url.getFile().toLowerCase().contains(os.name()))
						|| getDriverName().equals("IEDriverServer")
						|| (IS_OS_MAC
								&& url.getFile().toLowerCase().contains("osx")))
						&& !out.contains(url)) {
					out.add(url);
				}
			}
		}`

**Comment by bonigarcia** [comment]
Thanks for reporting. This is a regression due to the support of different binary names (i.e. a list instead a single String). It is solved for the next release (1.4.9).
